integrations/ultralytics-yolov5/README.md

```diff
@@ -15,57 +15,47 @@ limitations under the License.
-creating smaller and faster deployable versions.
-The techniques include, but are not limited to:
-- Pruning
-- Quantization
-- Sparse Transfer Learning
-
-After training, the model can be deployed with Neural Magic's DeepSparse Engine. The engine enables inference with GPU-class performance directly on your CPU.
-
-This integration enables spinning up one of the following end-to-end functionalities:
-- **Sparsification of YOLOv5 Models** - easily sparsify any of the YOLOV5 and YOLOV5-P6 models, from YOLOv5n to YOLOv5x models. 
-- **Sparse Transfer Learning** - fine-tune a sparse backbone model (or use one of our [sparse pre-trained models](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=detection&page=1)) on your own, private dataset.
+
+By integrating with robust training flows in the YOLOv5 repository, SparseML enables you to train inference-optimized sparse versions of YOLOv5 models on your dataset.
+
+There are two pathways:
+- **Sparse Transfer Learning** - fine-tune a pre-sparsified YOLOv5 checkpoint on your own dataset **[RECOMMENDED]**
+- **Sparsification from Sractch** - apply pruning and quantization to sparsify any of the YOLOv5 and YOLOv5-P6 models from scratch.
+
+Once trained, SparseML enables you to export models to the ONNX format, such that they can be deployed with DeepSparse for GPU-class performance on the CPU.
 
 ## Installation
 
-```pip install sparseml[torchvision]```
+Install with `pip`:
 
-Note: YOLOv5 will not immediately install with this command. Instead, a sparsification-compatible version of YOLOv5 will install on the first invocation of the YOLOv5 code in SparseML.
+```bash
+pip install sparseml[torchvision]
+```
+
+**Note**: YOLOv5 will not immediately install with this command. Instead, a sparsification-compatible version of YOLOv5 will install on the first invocation of the YOLOv5 code in SparseML.
 
 ## Tutorials
 
-- [Sparsifying YOLOv5 Using Recipes](https://github.com/neuralmagic/sparseml/blob/main/integrations/ultralytics-yolov5/tutorials/sparsifying_yolov5_using_recipes.md)
-- [Sparse Transfer Learning With YOLOv5](https://github.com/neuralmagic/sparseml/blob/main/integrations/ultralytics-yolov5/tutorials/yolov5_sparse_transfer_learning.md)
+- [Sparse Transfer Learning with the CLI](tutorials/sparse-transfer-learning.md) **[HIGHLY RECOMMENDED]**
+- [Sparsifying From Scratch with the CLI](tutorials/sparsification-from-scratch.md)
 
 ## Quick Tour
 
-### Recipes
+### SparseZoo
 
-Recipes encode the instructions and hyperparameters for sparsifying a model using modifiers to the training process.
-The modifiers can range from pruning and quantization to learning rate and weight decay.
-When appropriately combined, it becomes possible to create highly sparse and accurate models.
+Neural Magic has pre-sparsified each version of YOLOv5. These models can be deployed directly or can be fine-tuned onto custom dataset via sparse transfer learning. This
```

Comment:
These models can be deployed directly or fine-tuned onto custom datasets via sparse transfer learning